specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json

```diff
@@ -803,12 +803,12 @@
           "description": "Version of the guest configuration."
         },
         "contentUri": {
```

Comment:
Thanks for the commit. If it's possible, could you add some examples for the usage of these properties you changed?